packages/edit-site/src/components/global-styles/background-panel.js

```diff
@@ -8,6 +8,11 @@ import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
  */
 import { unlock } from '../../lock-unlock';
 
+// Initial control values where no block style is set.
+const BACKGROUND_DEFAULT_VALUES = {
+	backgroundSize: 'auto',
```

Comment:
This PR is useful regardless of the defaults we choose. 

See:

- https://github.com/WordPress/gutenberg/pull/60405

We want to display the default value in the control regardless of what it is.

`background-size` is technically not set at all, but the effect is the same as "Fixed" which is the label for `"background-size: auto"`